CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+## 2022-11-14 v0.13.0
+
+Release highlights:
+- A [Waku canary tool](https://github.com/status-im/nwaku/tree/2336522d7f478337237a5a4ec8c5702fb4babc7d/tools#waku-canary-tool) to check if nodes are reachable and what protocols they support.
+- Simplified configuration for store protocol. This [new guide](https://github.com/status-im/nwaku/blob/4e5318bfbb204bd1239c95472d7b84b6a326dd9d/docs/operators/how-to/configure-store.md) explains how to configure store from this release forward.
```

Comment:
I can see #1374 is part of this release. Perhaps it's worth adding as a highlight that `store` is no longer true by default?

Reply:
Added this as a "change" in [3cf5cb5](https://github.com/status-im/nwaku/pull/1378/commits/3cf5cb59e005e93dacd01ec4ccb846e4433f4a26). Concerned that adding it as a highlight would confuse people to think the default behaviour is now different, which it isn't (previously a node would also not have stored messages by default). The mounting of the protocol is more subtle and (probably) does not imply behavioural change for most operators.